posts/how-to-add-open-source-software-projects-to-sourcegraph-cloud.md

```diff
@@ -2,6 +2,7 @@
 title: How to add open source software projects to Sourcegraph cloud
 tags: [tutorial, open source, Sourcegraph]
 author: lisa-tagliaferri
+publicationDate: May 11, 2021
```

Comment:
I went through when PRs were merged to determine these dates, and will add to the README that that is what our approach is for date of publication

Reply:
This seems like a reasonable approach!